CHANGELOG.md

```diff
@@ -65,8 +69,6 @@ should change the heading of the (upcoming) version to include a major version b
 - New package created by refactoring and converting to Typescript the `validator.js` file from `core` into independent functions as well as a class that implements the new `ValidatorType` interface.
   - [#2693](https://github.com/rjsf-team/react-jsonschema-form/issues/2693).
 - Added support for customizing the options passed to the creation of the `ajv` instance.
-
-## @rjsf/validator-ajv6
```

Comment:
There were two of these sections, so I removed the duplicate to merge with the one above